client/app/templates/Components/Form/_checkboxgroup.html.twig

```diff
@@ -55,7 +55,10 @@
                     {% set attr = attr | merge({'data-aria-controls': choiceElement.vars.id ~ '-conditional'}) %}
                 {% endif %}
 
-                {{ form_widget(choiceElement, {'attr': attr}) }}
+                <div id="{{ element.vars.id }}">
```

Comment:
I think we should be putting the ID on the form widget, not on a div around it. Also note my previous comment: I think aria-described by should be on the form widget, not on the link to it.